CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # NVIDIA Container Toolkit Changelog
 
+### v1.16.0-rc.2
```

Comment:
Was this generated with `generate-changelog.sh`??? 🤓

Reply:
Yes, but edited.